data_summaries/EverythingLM.json

```diff
@@ -35,6 +45,10 @@
             {
                 "License": "MIT License",
                 "License URL": "https://huggingface.co/datasets/totally-not-an-llm/EverythingLM-data-V3"
+            },
+            {
+                "License": "OpenAI",
+                "License URL": "https://huggingface.co/datasets/totally-not-an-llm/EverythingLM-data-V3"
```

Comment:
Can you change this to the OAI URL? cc @shayne-longpre

Reply:
Sure, but in other PR's I saw people using the respective dataset's licence or URL itself. e.g. 

- https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/004761178ea7b9ea9c542d081296edc5e66bc314/data_summaries/Alpaca.json#L53
- https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/004761178ea7b9ea9c542d081296edc5e66bc314/data_summaries/GPTeacher.json#L51
- https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/004761178ea7b9ea9c542d081296edc5e66bc314/data_summaries/UltraChat.json#L57

Also, do you want me to use the specific OAI model's URL?

Reply:
I think this is fine -- we have the OAI URL encoded elsewhere that we can always link to. It's nice to see where they specify that they use OAI to generate the data so we can infer that the license is accurate.